nix/docker/k8s/cardano-nodes.yaml

```diff
@@ -0,0 +1,190 @@
+#
+# NAMESPACE=cardano
+# kubectl create namespace $NAMESPACE
+#
+
+# GCE Storage Class
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: cardano-standard-rwo
+provisioner: kubernetes.io/gce-pd
+parameters:
+  type: pd-standard
+  fstype: ext4
+  replication-type: none
+
+# Relay ###############################################################################################################
+
+# Relay Volume Claim
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: relay-data
+  labels:
+    app: cardano
+    pvc: relay-data
+spec:
+  storageClassName: cardano-standard-rwo
+  accessModes: [ ReadWriteOnce ]
+  resources:
+    requests:
+      storage: 10G
+
+# Relay Pod
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: relay
+  labels:
+    app: cardano
+    pod: relay
+spec:
+  containers:
+  - name: relay
+    image: inputoutput/cardano-node:dev
+    imagePullPolicy: Always
+    env:
+    - name: CARDANO_PUBLIC_IP
```

Comment:
You may consider using `envFrom` (https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#configure-all-key-value-pairs-in-a-configmap-as-container-environment-variables) to take all of the keys from a given config map and configure them as env vars to reduce repetition/improve readability.

Reply:
This also needs to work for [plain Docker and Docker Compose](https://tdiesler.gitbook.io/cardano/v/iohk/)